test/jdk/tools/jpackage/helpers/jdk/jpackage/test/Executor.java

```diff
@@ -457,15 +481,17 @@ public String getPrintableCommandLine() {
             exec = executablePath().toString();
         }
 
-        return String.format(format, printCommandLine(exec, args),
-                args.size() + 1);
+        var cmdline = Stream.of(prefixCommandLineArgs(), List.of(exec), args).flatMap(
+                List::stream).toList();
+
+        return String.format(format, printCommandLine(cmdline), cmdline.size() + 1);
```

Comment:
`cmdline.size() + 1` seems to be wrong. It should be `cmdline.size()`.
 ` + 1` was needed previously when was applied to the `args.size()` that didn't count the executable. Sorry, I oversaw this thing in my patch.

Reply:
Thanks, I understood why `+ 1` is not needed. I removed `+ 1`.